services/storage/tests/unit/test_rpc_handlers_simcore_s3.py

```diff
@@ -204,6 +210,7 @@ def short_dsm_cleaner_interval(monkeypatch: pytest.MonkeyPatch) -> int:
 async def test_copy_folders_from_valid_project_with_one_large_file(
     initialized_app: FastAPI,
     short_dsm_cleaner_interval: int,
+    with_storage_celery_worker_controller: TestWorkController,
```

Comment:
I would advise to put that fixture at the end of the fixtures list.
and why is it suddenly needed here?